keyboards/keebio/iris/keymaps/shinze/keymap.c

```diff
@@ -0,0 +1,173 @@
+#define BP_RALT  KC_RALT
+#define BP_WSH   MT(KC_LSHIFT, BP_W)
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [_BEPO] = LAYOUT(
+  //,------+--------+--------+--------+--------+--------.                    ,-------+--------+---------+--------+--------+---------.
+    KC_ESC , BP_DQOT, BP_LGIL, BP_RGIL, BP_LPRN, BP_RPRN,                     BP_AT  , BP_PLUS, BP_MINUS, BP_SLSH, BP_ASTR, BP_EQUAL,
+  //,------+--------+--------+--------+--------+--------.                    ,-------+--------+---------+--------+--------+---------.
+    BP_TAB , BP_B   , BP_ECUT, BP_P   , BP_O   , BP_EGRV,                     BP_DCRC, BP_V   , BP_D    , BP_L   , BP_J   , BP_Z ,
+  //,------+--------+--------+--------+--------+--------.                    ,-------+--------+---------+--------+--------+---------.
+    BP_WSH , BP_A   , BP_U   , BP_I   , BP_E   , BP_COMM,                     BP_C   , BP_T   , BP_S    , BP_R   , BP_N   , BP_M,
+  //,------+--------+--------+--------+--------+--------+---------. ,--------,-------+--------+---------+--------+--------+---------.
+    KC_LSHIFT,BP_AGRV, BP_Y   , BP_X   , BP_DOT , BP_K   , BP_RASE,   BP_RCTL, BP_APOS, BP_Q   , BP_G    , BP_H   , BP_F   , BP_CCED,
+  //,------+--------+--------+--------+--------+--------+---------. ,--------,-------+--------+---------+--------+--------+---------.
+                                BP_RALT, BP_RASE, BP_LOWR,          RSFT(BP_SPC), BP_ENT , BP_BSPC
+  ),
+
+  [_LOWER] = LAYOUT(
+  //,------+--------+--------+--------+--------+--------.                    ,-------+--------+---------+--------+--------+---------.
+    _______, _______, _______, _______, _______, _______,                     _______, _______, _______  , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------.                    ,-------+--------+---------+--------+--------+---------.
+    BP_DLR , BP_DQOT, BP_LGIL, BP_RGIL, BP_LPRN, BP_RPRN,                     BP_AT  , BP_PLUS, BP_MINUS, BP_SLSH, BP_ASTR, BP_EQUAL,
+  //,------+--------+--------+--------+--------+--------.                    ,-------+--------+---------+--------+--------+---------.
+    _______,_______ , _______ , _______ , KC_UP  ,_______  ,                  _______, _______ , _______ , _______, _______ ,BP_PERCENT,
+  //,------+--------+--------+--------+--------+--------+---------. ,--------,-------+--------+---------+--------+--------+---------.
+    _______, _______, _______ , KC_LEFT , KC_DOWN , KC_RIGHT,_______, _______,_______, _______ ,_______ , _______, _______ , _______,
+  //,------+--------+--------+--------+--------+--------+---------. ,--------,-------+--------+---------+--------+--------+---------.
+                                    BP_RALT, BP_RGUI, BP_LOWR,          SFT_T(BP_SPC), BP_ENT , BP_BSPC
+  //                               +--------+-------+---------.       ,--------------+--------+--------                                
+  ),
+
+  [_LOWER] = KC_KEYMAP(
```

Comment:
`KC_KEYMAP` no longer exists, and has been removed.

In general, we recommend against macros that prepend `KC_`, as they cause confusion.

But if you want to use it, you want:
```suggestion
  [_LOWER] = LAYOUT_kc
```